### ISSUE TITLE
Backups with no Data still shows in Longhorn and S3

### ISSUE THREAD

**Issue by vistalba** [longhorn/longhorn#597]
I deleted a volume around 25 days ago.
This volume has some backups. Yesterday I deleted all the backups from each device (manually) in the Longhorn UI.

Now, this volume is still in the "Backup" List but has no "Backup" behind. In S3 the Folder exists for example `backupstore/volumes/e9/ef/VOLUMENAME` and has a size of 10.88GB. 

How can I remove this stuff from my Backuplist and S3?

---
Version: `v0.5.0`

**Comment by yasker** [comment]
@vistalba That sounds like a bug. The data for the volume should be gone and the volume shouldn't show up in the backup list if there is no backup.

Can you show me the directory structure (and files) under `backupstore/volumes/e9/ef/VOLUMENAME`? Maybe the last backup cleanup was interrupted so the volume won't be removed.

(btw, seems we need to have a `delete backup volume` option. May want to file a separate issue.).

**Comment by vistalba** [comment]
@yasker 

First... there should be a option like `delete all backups of volume ...`

Don't know how to export this structure in AWS Webconsole.

There is a file `backupstore/volumes/e9/ef/VOLUMENAME/volume.cfg` and a folder `backupstore/volumes/e9/ef/VOLUMENAME/blocks`.

In the folder there are a lot of folders like:  `1b`, `1c`, `1d` ... `ff`. And in this folders there are again folders like `67`. `69` ... `f1`.

Is it safe to just delete the wohle folder with all subfolders and files ``backupstore/volumes/e9/ef/VOLUMENAME/` for all `VOLUMENAME` that matches Backups without any backup in LonghornUI or do I've to do some special steps?

**Comment by yasker** [comment]
@vistalba Yeah, that what I meant. We need an option to delete all the backups of a volume. I've filed #598 for this.

And yes, it's safe to delete that `VOLUMENAME` directory. We will try to reproduce the issue locally and keep this issue open to tracking the blocks leftover issue.

**Comment by vistalba** [comment]
@yasker Okay fine. Then I will delete the stuff there :) 
My cluster rancher cluster run into wired issues with creating new nodes so I decided to create a new one and restroe the volumes from there (because it's not a production environment).

I installed longhorn on the new cluster and restored some all of the used volumes. In this step I realized that I've some volumebackups that didn't exist anymore and decided to delete this backups. 
Total 2 volumes with 8 backups each around 20-25GB size.
